client/components/card/compact.jsx

```diff
@@ -3,14 +3,14 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	assign = require( 'lodash/assign' ),
-	classnames = require( 'classnames' );
+import React from 'react';
+import { assign } from 'lodash';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
-var Card = require( 'components/card' );
+import Card from 'components/card';
 
 module.exports = React.createClass( {
```

Comment:
We should be consistent in module syntax. Updated this to `export default` in 6729ba49faf0fa55bc807e52ee5682c470de88cf.